### ISSUE TITLE
ValueError: invalid literal for int() with base 10 if ForeignKey to_field is str

### ISSUE THREAD

**Issue by luzfcb** [treyhunner/django-simple-history#182]
to reproduce the problem, create the model class below.
After, run makemigrations, and migrate. Create a new Person and save then.
Include a HistoricalRecords() field and run  makemigrations, and migrate.
and finally, run populate_history --auto

```
from django.db import models
from django.conf import settings
from simple_history.models import HistoricalRecords


USER_MODEL = getattr(settings, 'AUTH_USER_MODEL', 'auth.User')

class Person(models.Model):
    content = models.TextField(blank=True)
    user = models.ForeignKey(to=USER_MODEL, to_field='username')

```
If this is expected behavior, it would be good to include it in Common Issues section of the documentation

ps. Sorry my ugly english.

**Comment by arski** [comment]
+1 on this, as in I have the exact same issue, in my case it occurs when trying to mommy.make a class like Person above

**Comment by macro1** [comment]
The bug is that `to_field` is not handled in our (transformation of foreign keys in models.py)[https://github.com/treyhunner/django-simple-history/blob/1.6.2/simple_history/models.py#L147]. If anyone would like to take a stab at fixing it and writing a test case to prevent it from being broken in the future, please do. Otherwise I may look at it later this month.

**Comment by arski** [comment]
Hey, I have a candidate fix but can't seem to figure out how to run tests on the module *duh* - could you instruct on the best approach please (tried manage.py test simple_history but that runs 0 tests.. simple_history.tests says module has no attribute 'tests').. must be missing something obvious.

**Comment by macro1** [comment]
@arski, the settings are in the `runtests.py` module, so you will need to run them through that.
```
python runtests.py
```
should do the trick.

**Comment by arski** [comment]
thanks a lot! raised a PR with tests etc., let me know what you think - thanks

**Comment by murataydos** [comment]
+1

**Comment by macro1** [comment]
Closing as it's merged. I'm going to look at some of the other issues and see if they can be addressed as well before releasing a new version though.

**Comment by arski** [comment]
Cool, thanks! Any issues in particular? Might be able to help in order to expedite a release :)

**Comment by macro1** [comment]
@arski, I'm hoping #185 will resolve... But at this point I think as long as Django >= 1.6 and django-simple-history >= 1.6.1 are used the database constraints should already be left off in migrations.

**Comment by macro1** [comment]
@luzfcb, @arski: Released fix as 1.6.3. Thanks for your help with this!

**Comment by arski** [comment]
yay, thanks! :)
